packages/pdfator-front/src/components/content/Options.vue

```diff
@@ -0,0 +1,54 @@
+<template>
+    <div>
+        <div class="options">
+          <input  
+            @input="$emit('input-output-file', $event.target.value)"
+            class="input" 
+            type="text" 
+            id="filename" 
+            placeholder="Filename"
+          >
+          <label 
```

Comment:
I personally like to wrap input in the label composent
```
<label>
  <input ...>
</label>
```

Reply:
i choosed this presentation to implement a float label pattern